src/config/source_map.yaml

```diff
@@ -1 +1,83 @@
 git:
+- name: openvino_master
+  automerge: 'yes'
+  mode: 'branch-to-branch'
+  src:
+    url: https://github.com/openvinotoolkit/openvino.git
+    branch: master
+  dest:
+    url: https://github.com/opendatahub-io/openvino.git
+    branch: master
+- name: openvino_contrib_master
+  automerge: 'yes'
+  mode: 'branch-to-branch'
+  src:
+    url: https://github.com/openvinotoolkit/openvino_contrib.git
+    branch: master
+  dest:
+    url: https://github.com/opendatahub-io/openvino_contrib.git
+    branch: master
+- name: openvino_model_server_main
+  automerge: 'yes'
+  mode: 'branch-to-branch'
+  src:
+    url: https://github.com/openvinotoolkit/model_server.git 
+    branch: main 
+  dest:
+    url: https://github.com/opendatahub-io/openvino_model_server.git 
+    branch: main
+- name: openvino_release_2023.3
+  automerge: 'yes'
+  mode: 'branch-to-branch'
+  src:
+    url: https://github.com/openvinotoolkit/openvino.git
+    branch: releases/2023/3
+  dest:
+    url: https://github.com/opendatahub-io/openvino.git
+    branch: 2023.3-release

```

Comment:
@Xaenalt @z103cb I thought decision was to have a more generic branch name so it could be a "continuous" release? Just wanted to confirm that having a specific release branch name would not impede the "continuity" of the release sync.

Reply:
@heyselbi let me check the diagram.

Reply:
I have changed the names to better match what is the diagram, I have only changed branch name `new` to `new_release`